sdlib/d/gc/tcache.d

```diff
@@ -93,10 +111,6 @@ public:
 			if (samePointerness && newSizeClass == oldSizeClass) {
 				return ptr;
```

Comment:
This is the only place in realloc that seems to require handling freeSpace, as far as i can tell, and this is also the only one left untouched.